<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Monitor Jakości Powietrza</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainVerticalLayout">
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Odswież</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cityFilterEdit">
        <property name="placeholderText">
         <string>Wpisz miasto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="filterButton">
        <property name="text">
         <string>Filtruj</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="addressInput">
        <property name="placeholderText">
         <string>Wpisz adres (np. Polanka 3, Poznań)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="radiusSpinBox">
        <property name="suffix">
         <string> km</string>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchNearbyButton">
        <property name="text">
         <string>Szukaj w okolicy</string>
        </property>
       </widget>
      </item>
      <item>
      <widget class="QToolBar" name="timeToolBar">
        <property name="movable">
         <bool>false</bool>
        </property>
        <widget class="QDateEdit" name="dateFromEdit"/>
        <widget class="QDateEdit" name="dateToEdit"/>
        <widget class="QPushButton" name="applyDateRangeButton">
         <property name="text">
          <string>Zastosuj</string>
         </property>
        </widget>
      </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="leftColumn">
        <item>
         <widget class="QListWidget" name="stationList"/>
        </item>
        <item>
         <widget class="QListWidget" name="sensorList"/>
        </item>
        <item>
         <widget class="QTextBrowser" name="logBrowser">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="rightColumn">
        <item>
        <widget class="QTableWidget" name="measurementTable">
        <property name="minimumSize">
            <size>
                <width>0</width>
                <height>100</height>
            </size>
        </property>
        <property name="maximumSize">
                <size>
                    <width>16777215</width>
                    <height>150</height>
                </size>
        </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
        </item>
        <item>
        <widget class="QTextBrowser" name="analysisBrowser">
            <property name="minimumSize">
                <size>
                    <width>0</width>
                    <height>100</height>
                </size>
            </property>
            <property name="maximumSize">
                    <size>
                        <width>16777215</width>
                        <height>150</height>
                    </size>
            </property>
        </widget>
        </item>
        <item>
         <widget class="QChartView" name="chartsBrowser"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
